src/api/Server.ts

```diff
@@ -79,6 +84,113 @@ export class SpacebarServer extends Server {
 		await Sentry.init(this.app);
 		WebAuthn.init();
 
+		let place = "";
+		this.app.use("*", (req: Request, res: Response, next) => {
+			if ((req.header("Referer") as string) != undefined) {
+				place = req.header("Referer") as string;
```

Comment:
Is this trying to autodiscover the voice endpoint to send to clients? Why? Just use the existing config value? This code is not very good.

Reply:
It is trying to auto discover endpoints, though it can be overwritten via env vars or db configs, though i didn't do the db config too much, this is for those who have 1 spacebar endpoint with access to an ip and a hostname, which is done on my private instance. It is also discovering for those who don't want to configure it, like for those just trying to test it out and see if they want to keep it.